### PR TITLE
add copy-to-clipboard buttons

### LINKED ISSUE #96: provide a copy+paste link to add a dub shield to a project
A few projects start to add backlink shields to code.dlang.org.
We should provide a common method to create such shields, that shows the latest version.

https://github.com/kiith-sa/dmarkdown
https://github.com/MartinNowak/qcheck

A nice extra feature would be to copy the `"qcheck": "~>0.1.0"` dependency line into the clipboard when clicking the link.

### PR DESCRIPTION
**Pull request by greenify**
adds [clipboard.js](https://clipboardjs.com/) to allow copy & paste for a package's dependency and shields.

### Package dependencies:

2016-03-21_22-42-24_scrot

### and for shields:

2016-03-21_22-43-18_scrot

update: fixes #96 and #140

### CODE REVIEW

**Comment by s-ludwig** [comment]
I would have added the shields to `my_packages.package.dt`, because I think that they are mostly useful for the authors themselves to link to their package. It might also make sense to put actual HTML `<a></a>` into the text box.

The clipboard additions in general and the dependency line change looks good! Having dub.sdl handled there was definitely overdue.

File: views/view_package.dt [hunk 1]
```diff
@@ -137,3 +144,24 @@ block body
 		- else
 			a(href='#{req.rootDir}packages/#{urlEncode(packageName)}/#{vs}')= vs
 
+	h2 Shields
+
+	#copy-shields
+		div
+			.img-wrapper
+				img(src="https://img.shields.io/dub/v/vibe-d.svg", alt="Dub version")
```
Comment (s-ludwig) [inline views/view_package.dt]: This should probably use `/#{packageName}.svg"`. Same for the one below.

**Comment by greenify** [comment]
> I would have added the shields to my_packages.package.dt, ...

Excellent point!

> It might also make sense to put actual HTML <a></a> into the text box.

I am not so sure on this point as shields are mostly used in the README.md and there markdown is used. Moreover bundling it into HTML tags requires mor chars and thus scrolling. 
On the other hand `<a>` tags work fine in markdown too, we already can fill out `alt` tags and for the majority of users it is more convenient.
(I changed it now)

**Comment by greenify** [comment]
Forgot to remove the shields from `view_package` when I moved them to `my_packages.package` - changed ;-)

**Comment by s-ludwig** [comment]
Okay, thanks, looks good to go!

**Pull request merged**